driver-adapters-wasm/neon-node-nextjs-edgemw/run.sh

```diff
@@ -12,5 +12,6 @@ wait_for_host() {
 
 pnpm install
 pnpm next dev --port 3000 &
+echo $! > server.pid
```

Comment:
Caused the second step for custom output tests to fail.

Reply:
What do you mean? It's hard to understand without context for me at the moment 🤔